source/d3d8to9.cpp

```diff
@@ -18,7 +18,10 @@ std::ofstream LOG;
 extern "C" Direct3D8 *WINAPI Direct3DCreate8(UINT SDKVersion)
 {
 #ifndef D3D8TO9NOLOG
-	LOG.open("d3d8.log", std::ios::trunc);
+	if (!LOG.is_open())
+	{
+		LOG.open("d3d8.log", std::ios::trunc);
+	}
 
 	if (!LOG.is_open())
 	{
```

Comment:
Maybe it's good to add an additional bool, so it won't spam MessageBoxes, every time user change the screen resolution and the log fails to open?